### PR TITLE
import so lib from nano not download it

### PR DESCRIPTION
**Pull request by zhanglirong1999**
- Modify some comment.
- Nano already has installed libturbojpeg.so.0.2.0 lib , so remove the additional installation.

### CODE REVIEW

**Comment by jason-dai** [comment]
Add description

File: python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py [hunk 1]
```diff
@@ -192,6 +192,8 @@ def create_dataloaders():
 if __name__ == "__main__":
     # get dataset
     model = MyLightningModule()
+    # BigDL-Nano can accelerate computer vision data pipelines
+    # by providing a drop-in replacement of torch_vision’s datasets and transforms
     train_loader, val_loader = create_dataloaders()
     # CV Data Pipelines
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py:198]: move the description below to above and combine with the description above

File: python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py [hunk 2]
```diff
@@ -15,26 +15,20 @@
 
 import torch
 import torchvision
-from turbojpeg import TurboJPEG, TJPF_GRAY
+from turbojpeg import TurboJPEG
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py:18]: shall we add built-in support for JPEP decode support in nano? @MeouSker77

Comment (MeouSker77) [reply python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py:18]: We already have `ImageFolder`, which has built-in support. But this Oxford Pet dataset has some gif pictures, so it cannot be used with `ImageFolder`. We may need to implement our Oxford Pet dataset, and warp these codes into our Oxford Pet dataset, then this example can use our Oxford Pet dataset directly.

**Comment by MeouSker77** [comment]
This PR will be closed due to a new version #5448

**Pull request closed without merging**